presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -499,3 +499,20 @@ See below for an example.
       ]
     }
 
+HDFS wire encryption
+--------------------
+
+In a Kerberized Hadoop cluster with enabled HDFS wire encryption you can enable
```

Comment:
Please ask someone else to review this commit. @mattsfuller ?

Reply:
I think @mattsfuller already reviewed this internally.